io.unobserved.espansogui.json

```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "io.unobserved.espansoGUI",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "espanso-gui",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=xdg-config/espanso"
```

Comment:
why not shipping espanso here instead?

Reply:
Sorry, not sure what you mean. If you mean packaging espanso to put on Flathub, I am not the maintainer and espanso requires permissions that may be beyond the limits of Flatpak.